tutorial/markdown/java/spring-boot.md

```diff
@@ -22,14 +22,55 @@ sdk_language:
 length: 30 Mins
 ---
 
-<!-- 
+<!--
   The name of this file does not need to be `tutorial-quickstart-java-springboot` because it is in the `tutorials/java/markdown` directory, so we can just call it `spring-boot`. The idea is that we can leave off `tutorial-quickstart` as a prefix.
 -->
 
 <!-- TODO:  Figure out how to add width to image size in try it now links -->
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=java-springboot-quickstart-repo&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/java-springboot-quickstart)
 
-## Prerequisites
+## Table of Contents
+
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Source Code](#source-code)
+  - [Install Dependencies](#install-dependencies)
+  - [Database Server Configuration](#database-server-configuration)
+  - [Environment Variables](#environment-variables)
+- [Running The Application](#running-the-application)
+  - [Directly on the machine](#directly-on-the-machine)
+  - [Docker](#docker)
+- [What We'll Cover](#what-well-cover)
+  - [Document Structure](#document-structure)
+  - [Code Organization](#code-organization)
+- [Data Model](#data-model)
+- [Let's Review the Code](#lets-review-the-code)
+  - [Integration Tests](#integration-tests)
+  - [Repository](#repository)
+  - [Model](#model)
+  - [Controller](#controller)
+  - [Service](#service)
+- [Route Specifications](#route-specifications)
+  - [GET Route](#get-route)
+  - [POST Route](#post-route)
+  - [PUT Route](#put-route)
+  - [DELETE Route](#delete-route)
+- [Route Workflow](#route-workflow)
+  - [GET Route Workflow](#get-route-workflow)
+  - [POST Route Workflow](#post-route-workflow)
+  - [PUT Route Workflow](#put-route-workflow)
+  - [DELETE Route Workflow](#delete-route-workflow)
+- [Custom SQL++ Queries](#custom-sql-queries)
+- [Running The Tests](#running-the-tests)
+- [Project Setup Notes](#project-setup-notes)
+- [Contributing](#contributing)
+- [Appendix](#appendix)
+  - [Extending API by Adding New Entity](#extending-api-by-adding-new-entity)
+  - [Running Self Managed Couchbase Cluster](#running-self-managed-couchbase-cluster)
+  - [Swagger Documentation](#swagger-documentation)
+
+## Getting Started
+
+### Prerequisites
 
 To run this prebuilt project, you will need:
```

Comment:
The tutorial should be Capella first. Please check the other tutorials to see how Couchbase Server is mentioned.

Reply:
updated from the other readmes